ui/src/components/terminal/TerminalDialog.vue

```diff
@@ -238,36 +245,22 @@ export default {
     },
 
     connectWithPassword() {
-      const params = Object.entries({
-        user: `${this.username}@${this.$props.uid}`,
-        passwd: encodeURIComponent(this.passwd),
-        cols: this.xterm.cols,
-        rows: this.xterm.rows,
-      })
-        .map(([k, v]) => `${k}=${v}`)
-        .join('&');
+      const passwd = encodeURIComponent(this.passwd);
```

Comment:
Is this `encodeURIComponent` already existent? if so, split the code rework and the fix into two different commits.